ckanext/ontario_theme/templates/internal/package/snippets/resources_list.html

```diff
@@ -17,9 +17,9 @@ <h3>{{ _('Data') }}</h3>
           {% for group in resources|selectattr("type","equalto","data")|groupby("data_range")|sort(attribute="grouper", reverse=True) %}
 
             {% if group.list[0].data_range != "N/A - N/A" %}
-              <h4>{{group.list[0].data_range}}</h4>
+              <h3 class="ontario-h3">{{group.list[0].data_range}}</h3>
```

Comment:
`ontario-h3` can be removed because the h2 tags don't have any overrides

Reply:
done